src/binary-reversal/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
Why do you need this? If you're using this for debugging, you can look into using the `Quokka.js` extension in vsCode.

Reply:
Thanks for the info Bond...noted